src/org/ods/services/OpenShiftService.groovy

```diff
@@ -171,4 +173,69 @@ class OpenShiftService {
       label: "Get phase of build ${buildId}"
     ).trim().toLowerCase()
   }
+  
+  public String getContainerForImage (String projectId, String rc, String image) {
+    script.sh(
+      script: """oc -n ${projectId} get rc ${rc} -o jsonpath='{.spec.template.spec.containers[?(contains .image "${image}")].name}'""",
```

Comment:
Missing `label` and `returnStdout`

Reply:
@michaelsauter  -we don't need a label - I get the rc by name ... stdOut seems the default

Reply:
Oh I meant `label` for the `script` (for better display in Blue Ocean). And while `returnStdout` seems to be the default, we are consistently setting it so I'd continue with it to avoid confusion.

Reply:
solved :)